app/presenters/export/csv/project_presenter.rb

```diff
@@ -264,4 +264,28 @@ def esfa_notes
 
     notes.map(&:body).join("\n\n")
   end
+
+  def other_contact_name
```

Comment:
Same here, makes me feel like perhaps the ContactsFetcherService should be responsible for this effort?